dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/DatadogCore.kt

```diff
@@ -598,5 +602,6 @@ internal class DatadogCore(
         internal val startupTimeNs: Long = System.nanoTime()
 
         internal const val OKHTTP_INTERCEPTOR_SAMPLE_RATE = "okhttp_interceptor_sample_rate"
+        internal const val OKHTTP_INTERCEPTOR_HOSTS_WITH_HEADER_TYPES = "okhttp_interceptor_hosts_with_header_types"
```

Comment:
minor: there is no hosts anymore

```suggestion
        internal const val OKHTTP_INTERCEPTOR_HEADER_TYPES = "okhttp_interceptor_header_types"
```

Reply:
fixed